rules/store/kv/store_test.go

```diff
@@ -522,6 +523,20 @@ func TestWriteRuleSetError(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestWriteRuleSetMismatchError(t *testing.T) {
```

Comment:
Should this be `StaleDataError`?

Reply:
Yes. This did not get update, my mistake.